### DISCUSSION TITLE
For You (Recomedation)

### DISCUSSION THREAD

**Discussion by Divakaran505** [fast4x/RiMusic#4507] [Ideas]
Dear RiMusic,

Thanks for making RiMusic app.

When I'm sad, happy, or just need to clear my mind, I love listening to music. I enjoy Tamil songs from all generations'

I have a request for you to consider regarding the "FOR YOU" Recommendations. Currently, there are 3 options. Please make the following changes:

Most Played Songs – This option is working well.
Random Songs – Please make this option play only random songs from all generations (70s to 2000s). Right now, it’s also suggesting most played songs.
Mixed Songs – This option should suggest a mix of random and most played songs for the user to listen to.

My other request is to add an option that allows users to block specific songs they don't want to hear. This feature would let me select and exclude songs I dislike from my playlist or recommendations
![1](https://github.com/user-attachments/assets/1e3f1e0b-e978-4614-add1-ee96b1e432a3)
![2](https://github.com/user-attachments/assets/34ff250e-b6a9-4e45-8b05-b61de3f8c858)

**Comment by frenchy82** [comment]
Really agree with this for the for you section especially for the mixed song
Currently I listen to several styles of music. It should offer me varied music in the style of my favorite music. 
Curently only one style of music is offered
